openssl/src/x509/mod.rs

```diff
@@ -649,6 +649,21 @@ impl X509Ref {
         }
     }
 
+    /// Returns this certificate's "alias". This field is populated by
+    /// OpenSSL in some contexts, such as PKCS12 parsing.
```

Comment:
Since OpenSSL doesn't document this API, it might be worth being a bit more specific: it contains the `friendlyName` attribute if it was present on the PKCS12.

As far as I know, this is the only way the library will set it. The x509 "app" has `-setalias` and the pcks12 app also sets it in some circumstances.

Reply:
Good idea.